ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/platform.j2

```diff
@@ -27,6 +27,9 @@ platform sand forwarding mode {{ platform.sand.forwarding_mode }}
 {%         if platform.sand.multicast_replication.default is arista.avd.defined %}
 platform sand multicast replication default {{ platform.sand.multicast_replication.default }}
 {%         endif %}
+{%         if platform.sand.mdb_profile is arista.avd.defined %}
+platform sand mdb profile {{ platform.sand.mdb_profile }}
```

Comment:
Did you check the order of commands on EOS? I would have expected these to be alphabetically sorted.

Reply:
I didn't, but testing on a 7280CR3K, it seems it's not really alpha sorted:

```
SJC13BDRLF01-A(config)#show run sec platform
platform sand lag mode 2048x8
platform sand multicast replication default ingress
platform sand mdb profile l3-xxl
```

Reply:
@ClausHolbechArista do you think this is OK?